framework/src/media/MediaRecorder.cpp

```diff
@@ -23,6 +23,8 @@
 namespace media {
 MediaRecorder::MediaRecorder() : mPMrImpl(new MediaRecorderImpl(*this))
 {
+	mId = (uint64_t)this << 32;
+	mId = mId | (uint64_t)mPMrImpl.get();
```

Comment:
how to make mId?
just wonder, What is this meaing?

how about 
`mId |= (uint64_t)mPMrImpl.get();`

Reply:
i did explain to you

Reply:
mId = left shift(this) << 32bit (this is pointer) + mPMrImpl.get() is also pointer

Reply:
In my opinion, just 'this' is enough. Isn' it?

Reply:
did not set a limit 
because the app does not know how to use this instance. 
This can be the same as creating and deleting instances.

Reply:
In addition,
 It is created by adding "this address" 32 bits and + "impl address" 
32 bits to ensure id's uniqueness.